src/cases/caseManager.ts

```diff
@@ -29,9 +29,10 @@ export class CaseManager {
 		this.#messageBus.subscribe(MessageKind.acceptCase, (message) =>
 			this.#onAcceptCaseMessage(message),
 		);
-		this.#messageBus.subscribe(MessageKind.jobsAccepted, (message) =>
-			this.#onJobsAcceptedOrJobsRejectedMessage(message),
-		);
+		// We should not delete the job after "apply". User may want to create PR for this job
```

Comment:
Then I suppose we need to still send this event and keep the job, but not display it? Actually we should keep it as an accepted job with some UI tag telling its state?

Reply:
@grzpab Yes, i think we need to keep it in the tree, but remove apply/dismiss buttons in accepted state, leaving only one option - create PR. Do i need to implement this in this PR?

Reply:
Added "accepted" state to the current PR